jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -406,10 +427,15 @@ def _on_refdata_change(self, msg={}):
             elif not len(viewer.data()):
                 self.link_type_selected = link_type_msg_to_trait['pixels']
 
+            if msg._data.label not in self.orientation.choices:
+                return
+
+            self.orientation.selected = msg._data.label

```

Comment:
why access `._data` instead of `.data` here?

Reply:
Only because I referred to the attrs in `__init__` without remembering the properties. 🤦‍♂️
```suggestion
            if msg.data.label not in self.orientation.choices:
                return

            self.orientation.selected = msg.data.label

```